<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CColorPicker</class>
 <widget class="QWidget" name="CColorPicker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>41</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSpinBox" name="pSpinBoxR">
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="pSpinBoxG">
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="pSpinBoxB">
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="pSpinBoxA">
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="pColorDisplay" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
